# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/clifford/mysqlcc/ssl_fix.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./mysqlcc.pro.in.orig	2003-08-06 00:31:41.000000000 +0200
+++ ./mysqlcc.pro.in	2003-08-06 00:32:16.000000000 +0200
@@ -27,7 +27,7 @@
 }
 
 unix {
-      LIBS        += @MYSQL_LIB@ 
+      LIBS        += @MYSQL_LIB@ -lssl
       INCLUDEPATH += @MYSQL_INCLUDE@
       CONFIG      += x11 
       OBJECTS_DIR  = tmp
